### PR TITLE
Web application using Streamlit

### PR DESCRIPTION
**Pull request by InputBlackBoxOutput**
Added information in README regarding web application built using Streamlit. 

Try it out at: https://inputblackboxoutput-streamlit-svg2mod-app-fszqih.streamlitapp.com/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,11 @@ It includes a modified version of [cjlano's python SVG parser and drawing module
 
 ```pip install svg2mod```
 
+#### Do not want to install the application?
```
Comment (Sodium-Hydrogen) [inline README.md:26]: Could this be changed to a weight 3 `###`?

File: README.md [hunk 2]
```diff
@@ -23,6 +23,11 @@ It includes a modified version of [cjlano's python SVG parser and drawing module
 
 ```pip install svg2mod```
 
+#### Do not want to install the application? 
+Use the web application instead: https://inputblackboxoutput-streamlit-svg2mod-app-fszqih.streamlitapp.com/
+
+Please request features and report bugs related to this web application at this [GitHub repository](https://github.com/InputBlackBoxOutput/streamlit-svg2mod)
+
```
Comment (Sodium-Hydrogen) [inline README.md:30]: Also would you be willing to add the other web based tool to this list?
[svg2mod.com](svg2mod.com)

**Comment by Sodium-Hydrogen** [comment]
Nice work!

File: README.md (hunk 1 above)
Comment (InputBlackBoxOutput) [reply README.md]: I have changed it to heading level 3

**Comment by Sodium-Hydrogen** [comment]
Would you be willing to fix the linting errors?

**Comment by InputBlackBoxOutput** [comment]
Hopefully, the last changes fixed the linting errors. BTW why do you have such sophisticated linting checks 🤔?

**Comment by Sodium-Hydrogen** [comment]
I use a linter at my day job and out of curiosity I ran it against this project and saw just how messy the code base was. I want the project to be approachable and for anyone to easily read the code and these checks force me and others to make sure what is committed is high quality.

Also github actions are free and easy to configure. They also are easy to turn on for forks so code can be linted and checked in a forked repo without having to update the pull request every time a linting check fails. Which makes github actions better than other systems I have had to use while contributing to other open source projects.

**Comment by InputBlackBoxOutput** [comment]
Hey @Sodium-Hydrogen, please fix the remaining linting errors. The remaining linting checks beat me 😅

**Pull request merged**